### PR TITLE
wind estimator start and add parameter to enable

### PR DESCRIPTION
**Pull request by dagar**

### CODE REVIEW

File: src/modules/wind_estimator/wind_estimator_main.cpp [hunk 1]
```diff
@@ -330,5 +344,12 @@ extern "C" __EXPORT int wind_estimator_main(int argc, char *argv[]);
 int
 wind_estimator_main(int argc, char *argv[])
 {
-	return WindEstimatorModule::main(argc, argv);
+	int32_t wind_estimator_enabled = 0;
+	param_get(param_find("WEST_EN"), &wind_estimator_enabled);
```
Comment (LorenzMeier) [inline src/modules/wind_estimator/wind_estimator_main.cpp]: We used to do this via `param compare` on the shell - do you want to change this on purpose?

Comment (dagar) [reply src/modules/wind_estimator/wind_estimator_main.cpp]: Yes I wanted it to work on more than just nuttx. I've also found our shell scripting to be quite brittle.

We should probably figure out an agreed path forward to converge on something consistent.

Comment (LorenzMeier) [reply src/modules/wind_estimator/wind_estimator_main.cpp]: That was my point. Fine for this.

Comment (bkueng) [reply src/modules/wind_estimator/wind_estimator_main.cpp]: @dagar I see your point, but the user/devel experience here is pretty bad:
- the wind estimator cannot be started from CLI w/o setting the param first (often needed during testing).
- trying to start the wind estimator with disabled param will simply not start it w/o any user message.

Thus I prefer the script approach with posix shell.

Comment (dagar) [reply src/modules/wind_estimator/wind_estimator_main.cpp]: I agree with all of that, but find those are minor compared to the bad user/devel experience we've created on non-nuttx.

I would propose we try to make a concrete plan for init unification. Even if we can't commit to it near term at least with an agreed on design in place we might be able to do some of it incrementally.

**Pull request merged**